teuthology/task/install/rpm.py

```diff
@@ -77,14 +77,15 @@ def _package_overrides(pkgs, os):
         result.append(pkg)
     return result
 
-def _zypper_addrepo(remote, repo_list):
```

Comment:
Please revert this change, it is not related to the issue of this PR and by the way name of signature were selected to correspond zypper long name command for adding repos, i.e. there no `zypper add repos` command, but `zypper addrepo` (long for `zypper ar`.